public_examples/kvtokencheck/checkKVTokens.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+## Setup the Environment Variables"
```

Comment:
Should you also mention the "DEBUG" env var?

Reply:
Updated Readme